runtime/string.inl

```diff
@@ -218,10 +218,10 @@ string::string(int64_t i) {
   const char *end = nullptr;
   if (static_cast<int64_t>(i32) == i) {
     p = create(STRLEN_INT32, true);
-    end = simd_int32_to_string(i32, p);
+    end = simd_int32_to_string(i32, p, STRLEN_INT32);
```

Comment:
Terminated null byte must be included in that length
```suggestion
    end = simd_int32_to_string(i32, p, STRLEN_INT32 + 1);
```

Reply:
I think i found a better solution
```suggestion
    end = simd_int32_to_string(i32, p, inner()->capacity);
```